### DISCUSSION TITLE
AJAX call limit?

### DISCUSSION THREAD

**Discussion by foxycode** [jiripudil/Naja#412] [Support] [answered]
Is there any AJAX call limit when using `naja.makeRequest()` or am I just misunderstood how this whole thing works?

I have two Naja extesions, both are working perfectly alone. When they are registered both, only second registered works. Both are sending request using `naja.makeRequest()` during initialize, but first one always ends up aborted. Everything works ok when I'm using raw `fetch()` requests.

**Comment by jiripudil** [comment, marked as answer]
Hi, yep, Naja by default makes sure that only one request is running at a time. You can disable this behaviour or even define separate unique groups: https://naja.js.org/#/extensions-default?id=uniqueextension

**Comment by foxycode** [comment]
Perfect, thanks :-)
